tst/testinstall/direct_factors.tst

```diff
@@ -49,6 +51,8 @@ gap> List(DirectFactorsOfGroup(G), IdGroup);
 [ [ 8, 3 ], [ 8, 3 ] ]
 gap> List(DirectFactorsOfGroup(SmallGroup(64,226):useKN), IdGroup);
 [ [ 8, 3 ], [ 8, 3 ] ]
+gap> List(DirectFactorsOfGroupKN(SmallGroup(8,5)), IdGroup);
+[ [ 2, 1 ], [ 2, 1 ], [ 2, 1 ] ]
```

Comment:
TBH, I was confused when I've seen `DirectFactorsOfGroupKN` and assumed that I've missed that we have introduced some other abbreviation like "NC". Then I figured out that this stands for "Kayal-Nezhmetdinov". Maybe I'd prefer `DirectFactorsOfGroupByKN` more.